tinync/tinync.c

```diff
@@ -87,47 +99,56 @@ struct cr {
 
 typedef cr_queue(uint8_t, 4096) byte_queue_t;
 
-static void stdin_loop(struct cr *o, byte_queue_t *out)
+typedef struct {
+    int fd;
+    byte_queue_t *queue;
+} sock_w_loc_t;
```

Comment:
Why did you call it `sock_w_loc_t`?

Reply:
`sock_w_loc_t` stands for "socket write local", since this structure aims to hold local variables referenced in coroutine `socket_write_loop`.

Reply:
How about `conn_ctx_t` or `conn_data_t`? Where `conn` implies "connections."

Reply:
I think `conn_data_t` is more reasonable than `sock_w_loc_t`, I'll modify it.